tls/s2n_connection.c

```diff
@@ -164,7 +164,7 @@ struct s2n_connection *s2n_connection_new(s2n_mode mode)
         }
         /* S2N does not have it's own x509 Certificate parser, so Client Mode should only be used for testing purposes.
```

Comment:
Should we modify this comment?

Reply:
modify to “we need to do more testing on our x.509 code. Untill then use with caution”

How does that sound?